linker/shared/src/test/scala/org/scalajs/linker/OptimizerTest.scala

```diff
@@ -100,13 +100,15 @@ class OptimizerTest {
 
     for (linkingUnit <- linkToLinkingUnit(classDefs, MainTestModuleInitializers)) yield {
       val linkedClass = linkingUnit.classDefs.find(_.encodedName == MainTestClassDefEncodedName).get
+      val WitnessMethodName = newMethodName("witness__O")
+      val CloneMethodName = newMethodName("clone__O")
       linkedClass.hasNot("any call to Foo.witness()") {
-        case Apply(_, receiver, Ident("witness__O", _), _) =>
+        case Apply(_, receiver, MethodIdent(WitnessMethodName, _), _) =>
```

Comment:
I take it the implicit conversions from `TestIRBuilder` do not work here?

Reply:
No they don't, because we are in a pattern.